src/sv-pipeline/scripts/vcf_qc/compare_callsets.sh

```diff
@@ -141,8 +140,7 @@ if [ ${CARRIER} == 1 ]; then
   mv ${OVRTMP}/set1.bed2 ${OVRTMP}/set1.bed
 fi
 #Unzip & format SET2, if gzipped
-if [ $( file ${SET2} | fgrep " gzip " | wc -l ) -gt 0 ] || \
-   [ $( echo ${SET2} | awk -v FS="." '{ if ($NF ~ /gz|bgz/) print "TRUE" }' ) ]; then
+if [ $( echo ${SET2} | awk -v FS="." '{ if ($NF ~ /gz|bgz/) print "TRUE" }' ) ]; then
```

Comment:
This check is less "stringent" than using `file` since the updated code will pass if you had only renamed the file and added `.gz` extension to it, while the previous checks the file header which is more reliable. However, given the issues caused by `file`, I think it is still reasonable to check only the extension, but treat with a bit of caution :)